USAGE_android.md

```diff
@@ -102,8 +213,135 @@ It may also be set as a post attach command in the project configuration:
 * Enter the `process handle SIGSEGV -n true -p true -s false` command
 * Click "Apply"
 
+
+### To Root Or Not To Root
+
+Rooting the Android device for capture is not required if you are attempting
+to capture an application you can build in Debug mode and associate it with the
```

Comment:
I think it works for any application marked debuggable - you don't have to build it yourself but that is one way to make sure it's marked.  Could you include a link to the Android docs on where the manifest debuggable flag is?  I think this might be it: https://developer.android.com/guide/topics/manifest/application-element#debug

Reply:
How about this language?

----

Rooting the Android device for capture is not required if you are attempting
to capture an application that is built in Debug mode.
This is especially true if you can build in Debug mode and associate it with the
GfxReconstruct layer during the build.

One of the ways to set the application to debug requires modifying the
application Manifest file and setting the `android:debuggable` flag to true
before rebuilding.
Refer to the following info:
https://developer.android.com/guide/topics/manifest/application-element#debug

Alternatively, if you are building using Gradle, then make sure that the
application's `build.gradle` contains the `debuggable` flag under the
appropriate "buildTypes" section:

```text
debug {
   debuggable true
}
```

Reply:
LGTM